src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -589,7 +589,7 @@ private function parseDefinition(string $id, array|string|null $service, string
                     } else {
                         $method = $call[0];
                         $args = $call[1] ?? [];
-                        $returnsClone = $call[2] ?? false;
+                        $returnsClone = isset($call[2]) && $call[2];
```

Comment:
Same for those 2 cases. My expectation here is that we always expected a boolean Yaml value (or nothing) in such case, which would not trigger the deprecation with the existing code.
If we have some tests using non-boolean values, we might want to fix those tests instead (and maybe trigger a deprecation in Symfony for projects using a non-boolean value).